projects/cocoapods.org/package.yml

```diff
@@ -34,4 +37,6 @@ test:
   env:
     # some of our test containers are raw and thus run as root
     COCOAPODS_ALLOW_ROOT: 1
+    linux:
+      LANG: en_US.UTF-8
```

Comment:
This is surfaced from the underlying environment. If it is not set on Linux then the builders should be modified. Since I don’t like relying on the underlying env, brewkit should set it.